store/postgres/src/relational.rs

```diff
@@ -554,18 +565,15 @@ impl Layout {
 
         for entity_data in inserts_or_updates.into_iter() {
             let entity_type = entity_data.entity_type();
-            let mut data: Entity = entity_data.deserialize_with_layout(self, None, false)?;
+            let data: Entity = entity_data.deserialize_with_layout(self, None, true)?;
             let entity_id = Word::from(data.id().expect("Invalid ID for entity."));
             processed_entities.insert((entity_type.clone(), entity_id.clone()));
 
-            // `__typename` is not a real field.
-            data.remove("__typename")
```

Comment:
is this ok to remove now? I seem to remember this being added (by Eva?) because this was causing some issues?

Reply:
This was just a refactor, a couple lines above the last parameter to `deserialize_with_layout` was changed to `true` so that the typename is already removed.